redbiom/_requests.py

```diff
@@ -91,18 +96,27 @@ def f(context, cmd, data):
     return f
 
 
-def make_script_exec(config):
+def make_script_exec(config, post=False):
```

Comment:
As far as I can tell there is no use in the code for `post=True`, right? Should we remove that option?

Reply:
I think that's correct -- good call. This was added in when screwing around with posts which behave a little different with webdis